packages/helper-module-context/test/index.js

```diff
@@ -1,30 +1,25 @@
 const { assert } = require("chai");
-const { parse } = require("@webassemblyjs/wast-parser");
 
 const { moduleContextFromModuleAST } = require("../lib");
 
-const contextFromWast = wast => moduleContextFromModuleAST(parse(wast).body[0]);
+const contextFromWast = ast => moduleContextFromModuleAST(JSON.parse(ast));
 
 describe("module context", () => {
   describe("start segment", () => {
     it("should return the start function offset", () => {
-      const context = contextFromWast(`
-        (module
-          (func)
-          (func)
-          (start 1)
-        )
-      `);
+      const context = contextFromWast(
+        '{"type":"Module","id":null,"fields":[{"type":"Func","name":{"type":"Identifier","value":"func_0","raw":""},"signature":{"type":"Signature","params":[],"results":[]},"body":[],"loc":{"start":{"line":3,"column":9},"end":{"line":3,"column":12}}},{"type":"Func","name":{"type":"Identifier","value":"func_1","raw":""},"signature":{"type":"Signature","params":[],"results":[]},"body":[],"loc":{"start":{"line":4,"column":9},"end":{"line":4,"column":12}}},{"type":"Start","index":{"type":"NumberLiteral","value":1,"raw":"1"},"loc":{"start":{"line":5,"column":9},"end":{"line":5,"column":15}}}],"loc":{"start":{"line":2,"column":7},"end":{"line":6,"column":5}}}'
```

Comment:
I think it was more readable with the Wast format. I'm wondering why you needed to change it?

Reply:
Ah for the devdep? That's fine